requirements/dev.txt

```diff
@@ -2,30 +2,30 @@
 
 # lint and formatting tools
 black==19.10b0
-flake8==3.9.2
```

Comment:
should we hold the lint and formatting tools back? I think if you change these, the "make format" command may reformat out code, does it not?